examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -364,14 +369,36 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 	}
 
-	getInstanceIndex( shaderStage = this.shaderStage ) {
+	getBuiltin( name, property, type, shaderStage = this.shaderStage ) {
 
-		this.builtins.add( 'instance_index' );
+		this.builtins[ shaderStage ].add( {
```

Comment:
I think some form of search should be implemented... Otherwise something like `set.add( { name: 1, property: 2 } ).add( { name: 1, property: 2 } )` would duplicate elements, because the objects are not equal.